pkg/plugins/manager/client/client.go

```diff
@@ -30,6 +31,11 @@ var (
 	errNilSender  = errors.New("sender cannot be nil")
 )
 
+var managedGrpcErrors = []errutil.Base{
```

Comment:
Can we maybe update this to `passthroughErrors` and add others errors like `plugins.ErrMethodNotImplemented`, and `plugins.ErrPluginUnavailable` - IE all the errors in which we "passthrough". Would be helpful to add a comment above to explain.

Reply:
https://github.com/grafana/grafana/pull/102372/commits/a3d001cc78b453b6a9fe03dd7de7f70af05bcbf4